capacity4more/modules/c4m/features_og/c4m_features_og_wiki/c4m_features_og_wiki.module

```diff
@@ -106,7 +106,8 @@ function c4m_features_og_wiki_block_view_menu_c4m_og_menu_alter(&$data, $block)
   $wiki['#title'] = t('Wiki');
 
   // Set a custom class for the WIKI link for easy theming.
-  $wiki['#attributes']['class'][] = 'wiki';
+  $wiki['#attributes']['class'] = array('wiki');
```

Comment:
Don't know why, but the link has "first" and "last" classes set, while it is just a menu link with lots of adjacent links.